### DISCUSSION TITLE
Bug: In GH Projects `@next` iteration are incorrect

### DISCUSSION THREAD

**Discussion by florius0** [community/community#20108] [Projects and Issues] [answered]
When there is no current iteration according to iteration field settings, `@next` is equal to the second iteration in list, but should be the first

**Comment by Neudrino** [comment, marked as answer]
There are several issues describing the same problem.

The **most up-voted** one is:
- https://github.com/github-community/community/discussions/20101

What was thought to be a **fix** was mentioned to be out for testing in:
- https://github.com/github-community/community/discussions/20101#discussioncomment-3149906

The others are:
- https://github.com/github-community/community/discussions/20217
- https://github.com/github-community/community/discussions/20032
- https://github.com/github-community/community/discussions/20108
- https://github.com/github-community/community/discussions/18834

**Comment by 2percentsilk** [comment]
Based on testing from the community and team, we believe we have fixed this bug. Thanks to everyone for reporting it!
